<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBDialogLoad</class>
 <widget class="QDialog" name="HBDialogLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/logo.ico</normaloff>assets/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:&quot;Roboto&quot;;
font-size:12;
font-weight: 600;
}

#widgetMain{
background-color: rgba(123, 197, 174, 0.95);
border-radius: 6px;
}

#textFilePath{
  background-color:rgba(209, 237, 225, 0.95);
  border-radius: 6px;
}

#widgetMain .QPushButton {
color: #FFFFFF;
background-color: #028C6A;
border-radius: 5px;

border-width: 2px;
border-style: solid;
border-color: #028C6A;
/*text-align:left;*/
padding:0px 8px;
}

#widgetMain .QPushButton:hover {
color: #003E19 ;
background-color: #7BC5AE;
border-width: 3px;
}

#textFilePath {
color: #000000 ;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>28</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>28</number>
      </property>
      <item>
       <widget class="Line" name="lineTop">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>36</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textFilePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1440</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="placeholderText">
         <string>Enter File Path...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetBtns" native="true">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>14</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnLoad">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>36</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelect">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>36</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>36</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1440</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
